auslib/blobs/apprelease.py

```diff
@@ -908,6 +917,62 @@ def getReferencedReleases(self):
         return referencedReleases
 
 
+class ProofXMLMixin(object):
+
+    def _getAdditionalPatchAttributes(self, patch):
+        additionalPatchAttributes = {}
+
+        if 'proof' in patch:
+            additionalPatchAttributes['proof'] = patch["proof"]
+
+        return additionalPatchAttributes
+
+
+class ReleaseBlobV8(ProofXMLMixin, ReleaseBlobBase, NewStyleVersionsMixin, MultipleUpdatesXMLMixin, UnifiedFileUrlsMixin):
+    """
+
+    Changes from ReleaseBlobV7:
+        * Adds support for ProofXMLMixin (placed as first parameter for inheritance preference)
+
+    For further information:
+        * https://bugzilla.mozilla.org/show_bug.cgi?id=1384296
+
+    """
+    jsonschema = "apprelease-v8.yml"
+
+    # for the benefit of get*XML
+    optional_ = ('showPrompt', 'showNeverForVersion',
+                 'actions', 'openURL', 'notificationURL', 'alertURL',
+                 'promptWaitTime', 'backgroundInterval',
+                 'transparencyCert', 'SCT1', 'SCT2')
```

Comment:
It looks like these still need to be added to the jsonschema + tests. Also, is there a bug or document that describes what we want the update response to look like?